README.md

```diff
@@ -1,22 +1,67 @@
-RedisMemo
-=========
-Caching is made easy.
+<img align="left" src="https://github.com/chanzuckerberg/redis-memo/blob/main/docs/images/icon.png?raw=true" alt="drawing" width="120"/>
+
+# RedisMemo
 
 [![Gem Version](https://badge.fury.io/rb/redis-memo.svg)](https://badge.fury.io/rb/redis-memo)
 [![Build Status](https://travis-ci.com/chanzuckerberg/redis-memo.svg?branch=main)](https://travis-ci.com/chanzuckerberg/redis-memo)
 [![codecov](https://codecov.io/gh/chanzuckerberg/redis-memo/branch/main/graph/badge.svg?token=XG83PSWPG0)](https://codecov.io/gh/chanzuckerberg/redis-memo)
 
-A Redis-based [version-addressable](https://github.com/chanzuckerberg/redis-memo/wiki/Version-Addressable) caching system. Memoize pure functions, aggregated database queries, and 3rd party API calls.
 
-## Getting Started
-### Add to your Gemfile
+## Add to your Gemfile
 ```ruby
 # -- Gemfile --
 gem 'redis-memo'
 ```
+## Usage
+### Cache simple ActiveRecord queries
+Add two lines to the `User` model:
+```ruby
+class User < ApplicationRecord
+  extend RedisMemo::MemoizeQuery
+  memoize_table_column :id
+end
+```
+
+`SELECT "users".* FROM "users" WHERE "users"."id" = $1` queries will **load the data from Redis** instead of the database:
+```
+[1] pry(main)> Post.last.author
+  Post Load (0.5ms)  SELECT "posts".* FROM "posts" ORDER BY "posts"."id" DESC LIMIT $1
+[Redis] User Load SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2
+[Redis] command=MGET args="RedisMemo::Memoizable:wBHc40/aONKsqhl6C51RyF2RhRM=" "RedisMemo::Memoizable:fMu973somRtsGSPlWfQjq0F8yh0=" "RedisMemo::Memoizable:xjlaWFZ6PPfdd8hCQ2OjJi6i0hw="
+[Redis] call_time=0.54 ms
+[Redis] command=MGET args="RedisMemo:SELECT \"users\".* FROM \"users\" WHERE \"users\".\"id\" = ? LIMIT ?::P+HaeUnujDi9eH7jZfkTzWuv6CA="
+[Redis] call_time=0.44 ms
+=> #<User id: 1>
+```
+Learn more [here](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord).
+
+### Cache aggregated ActiveRecord queries
+Some computation might depend on multiple database records, for example:
```

Comment:
```suggestion
Some computation might depend on multiple database records, for example, the `display_title` below does not only depend on the post, but also depends on the author of the post, we'll need to add two lines with `depends_on`:
```
I think adding some why depends_on these two things will be easier to understand.

Reply:
hmm, to me this is a bit too verbose. the code pretty much says the same thing with less words